<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab</class>
 <widget class="QWidget" name="Tab">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="PlainTextEdit" name="plainTextEdit">
     <property name="placeholderText">
      <string>Paste URLs here...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="donationLinkLabel">
     <property name="toolTip">
      <string>This app is the result of countless hours of development by Marko PejiÄ‡. If it's useful to you, please consider supporting its development with a donation.&lt;br&gt;Thank you!</string>
     </property>
     <property name="text">
      <string>&lt;a href=&quot;https://downloader.markopejic.com/donate&quot;&gt;Donate&lt;/a&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QFormLayout" name="downloadOptionsLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldGrowthPolicy::FieldsStayAtSizeHint</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::RowWrapPolicy::WrapLongRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="formatLabel">
       <property name="text">
        <string>Format:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ComboBox" name="formatComboBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Format Options&lt;br&gt;(Alt+F)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="qualityLabel">
       <property name="text">
        <string>Quality:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ComboBox" name="qualityComboBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Quality Options&lt;br&gt;(Alt+Q)</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="placeholderText">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="subtitlesLabel">
       <property name="text">
        <string>Subtitles:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ComboBox" name="subtitlesComboBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Subtitle Options&lt;br&gt;(Alt+S)</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="placeholderText">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="downloadFolderLabel">
       <property name="text">
        <string>Download Folder:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="downloadFolderLayout">
       <item>
        <widget class="QLabel" name="downloadFolderIndicatorLabel">
         <property name="text">
          <string>None</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setDownloadFolderButton">
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Alt+C</string>
         </property>
         <property name="text">
          <string>&amp;Change...</string>
         </property>
         <property name="icon">
          <iconset theme="folder-open"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="cropThumbnailsCheckBoxLayout">
     <item>
      <spacer name="horizontalSpacer3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cropThumbnailsCheckBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Useful when downloading music&lt;br&gt;(Alt+T)</string>
       </property>
       <property name="text">
        <string>Crop &amp;thumbnails / album arts to square shape</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="embedSubtitlesCheckBoxLayout">
     <item>
      <spacer name="horizontalSpacer5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="embedSubtitlesCheckBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Embed subtitles directly into audio/video files&lt;br&gt;(Alt+E)</string>
       </property>
       <property name="text">
        <string>&amp;Embed subtitles</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer6">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer7">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionButtonsLayout">
     <item>
      <spacer name="horizontalSpacer8">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="LargePushButton" name="dataPullButton">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Get subtitle and quality options for all URLs&lt;br&gt;(Alt+P)</string>
       </property>
       <property name="text">
        <string>&amp;Pull Data</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh"/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LargePushButton" name="downloadButton">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Alt+D</string>
       </property>
       <property name="text">
        <string>&amp;Download</string>
       </property>
       <property name="icon">
        <iconset theme="emblem-downloads"/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer9">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer10">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statusIndicatorsLayout">
     <item>
      <spacer name="horizontalSpacer11">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusIconLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer12">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>3</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer13">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>..custom_widgets.combobox</header>
  </customwidget>
  <customwidget>
   <class>PlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>media_downloader.custom_widgets.plaintextedit</header>
  </customwidget>
  <customwidget>
   <class>LargePushButton</class>
   <extends>QPushButton</extends>
   <header>media_downloader.custom_widgets.largepushbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
